### PR TITLE
#6111 feat(refactor): regulates use of react context

### PR DESCRIPTION
**Pull request by chriddell**
Resolves https://github.com/wellcometrust/corporate/issues/6111

- Moves _SearchPaneContext_ to component level

Ultimately we may want to manage Context differently (separate folder? within components?) but this brings all current uses of Context in to line with each other.

### CODE REVIEW

File: src/components/SearchPane/SearchPaneForm.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import Button from 'Button/Button';
+import Button from 'Button';
```
Comment (creido-welly) [inline src/components/SearchPane/SearchPaneForm.tsx:3]: Surely this still needs to be `Button/Button` ?

Comment (chriddell) [reply src/components/SearchPane/SearchPaneForm.tsx:3]: Nope, can be either - Button has an index.ts file so this works... however it does beg the question "which way is better?" (as both methods are used in this repo 😫).

I would argue the above is better, as it means if the main export in Button changes (to e.g. ButtonView) we can just update the Button/index.ts file to export that, and not have to update every component which uses the Button component

Comment (creido-welly) [reply src/components/SearchPane/SearchPaneForm.tsx:3]: As discussed, if it's working in situ then all good. We can address the wider question of default vs named export in `src/index.ts` another time if need be

**Review by creido-welly — APPROVED** [review]

File: src/components/SearchPane/SearchPaneForm.tsx (hunk 1 above)
Comment (chriddell) [reply src/components/SearchPane/SearchPaneForm.tsx:3]: Will fix conflicts and merge today

**Pull request merged**